packages/tracy-client/tracy-client.0.1/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Client bindings to the Tracy profiler (v0.9.1)"
+maintainer: ["Simon Cruanes"]
+authors: ["Bartosz Taudul" "Simon Cruanes"]
+license: "BSD-3-Clause"
+homepage: "https://github.com/imandra-ai/ocaml-tracy"
+bug-reports: "https://github.com/imandra-ai/ocaml-tracy/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.08"}
+  "trace"
+  "odoc" {with-doc}
+]
+depexts: [
+  ["linux-headers"] {os-distribution = "alpine"}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/imandra-ai/ocaml-tracy.git"
+url {
+  src:
+    "https://github.com/imandra-ai/ocaml-tracy/releases/download/v0.1/tracy-client-0.1.tbz"
```

Comment:
I tried to download the tarball, and it seems that it contains two identical physical copies of tracy distribution (one under vendors/tracy, and another under src/bindings/tracy)

This is probably a non-blocker but imho worthy to look into.